hive/src/main/scala/io/delta/hive/DeltaInputFormat.scala

```diff
@@ -74,12 +74,7 @@ class DeltaInputFormat(realInput: ParquetInputFormat[ArrayWritable])
       reporter: Reporter): RecordReader[NullWritable, ArrayWritable] = {
     split match {
       case deltaSplit: DeltaInputSplit =>
-        if (Utilities.getIsVectorized(job)) {
```

Comment:
@YannByron do you know if it's okay to use non vectorized reader when the config is enabled? Hive 3 enables it by default. I removed this check but didn't see any issues in tests.

Reply:
@zsxwing I agree to remove this check.
At Hive's point, should enable it by default to make more users and scenes full use of this.
IMO, there must be some bugs In some extreme situations, but this‘s not about delta.

Reply:
@zsxwing I agree to remove this check.
At Hive's point, should enable it by default to make more users and scenes full use of this.
IMO, there must be some bugs In some extreme situations, but this‘s not about delta.